website/source/docs/job-specification/job.html.md

```diff
@@ -87,6 +87,9 @@ job "docs" {
 - `update` <code>([Update][update]: nil)</code> - Specifies the task's update
   strategy. When omitted, rolling updates are disabled.
 
+- `vault` <code>([Vault][]: nil)</code> - Specifies the configuration options
```

Comment:
"Specifies the configuration options for connecting the tasks in this group to Vault." ->
Specifies the set of Vault policies required by all tasks in the job